app/src/components/tables/DeploysTable/DeploysTable.test.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import { Table } from 'src/components';
+import {
+  getMockDeploysTableFooter,
+  getMockDeploysTableHeader,
+  getMockProcessedSidecarDeploy,
+  mockDeploysTableColumns,
+  mockDeploysTableOptions,
+} from 'src/mocks/mock-deploys-table';
+import { formatTimeAgo, standardizeNumber, truncateHash } from 'src/utils';
+import { render } from '../../../test-utils';
+
+const processedSidecarDeploys = getMockProcessedSidecarDeploy();
+const mockDeploysTableHeader = getMockDeploysTableHeader();
+const mockDeploysTableFooter = getMockDeploysTableFooter();
+const totalPages = Math.ceil(
+  processedSidecarDeploys.length / mockDeploysTableOptions.pagination.pageSize,
+);
+
+describe('DeploysTable', () => {
+  beforeEach(() => {
+    render(
+      <Table
+        header={mockDeploysTableHeader}
+        columns={mockDeploysTableColumns}
+        data={processedSidecarDeploys}
+        footer={mockDeploysTableFooter}
+        currentPageSize={mockDeploysTableOptions.pagination.pageSize}
+        placeholderData={{}}
+        isLastPage={totalPages === mockDeploysTableOptions.pagination.pageSize}
+      />,
+    );
+  });
+
+  it('should render Deploys Table', () => {
+    const deploysTableHeader = screen.getByTestId('deploys-table-header');
+    const deploysBaseTable = screen.getByTestId('base-table');
+    const deploysTableFooter = screen.getByTestId('deploys-table-footer');
+
+    expect(deploysTableHeader).toBeInTheDocument();
+    expect(deploysBaseTable).toBeInTheDocument();
+    expect(deploysTableFooter).toBeInTheDocument();
+  });
+
+  it('should render Total Rows', () => {
+    const totalRows = screen.getByTestId('total-rows');
+    expect(totalRows).toHaveTextContent('1 total rows');
```

Comment:
Can we do something more dynamic here? Maybe `expect(totalRows).toHaveTextContent(`${totalRows.length} total rows`);`. That way if we ever update the data length it will automatically update here

Reply:
Updated!